src/update.ts

```diff
@@ -110,7 +119,7 @@ async function updateMonorepo(
   newVersion: string,
   versionDiff: SemverReleaseType,
   rootManifest: Pick<
```

Comment:
Could this use the `MonorepoPackageManifest` type directly instead, rather than picking just those two properties?